command/volume_snapshot_list.go

```diff
@@ -34,8 +35,8 @@ General Options:
 
 List Options:
 
-  -plugin: Display only snapshots managed by a particular plugin. By default
-   this command will query all plugins for their snapshots.
```

Comment:
This "by default" behavior never worked and never _could_ work, so I'm just calling this a mulligan for the CSI beta and not a backcompat issue we need to worry about. 😀

Reply:
:+1: